contracts/utils/TimeLock.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { ITimeLock } from '../interfaces/ITimeLock.sol';
+
+contract TimeLock is ITimeLock {
```

Comment:
Need to add unit tests for the `TimeLock` contract.

Reply:
Unit tests PR [here](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/pull/42)